### PR TITLE
Set custom file headers

### LINKED ISSUE #56: chore: create custom Xcode headers
**Describe the bug**
Xcode is inserting default headers into each source code file such as:
//
//  Members.swift
//  mentorship ios
//
//  Created by Yugantar Jain on 07/06/20.
//  Copyright © 2020 Yugantar Jain. All rights reserved.
//

**To Reproduce**
1. Open the project in Xcode
2. Create a new Swift file.
3. Observe the file headers.

**Desired behavior**
The following headers are preferred:

//
//  Members.swift
//
//  Created for the AnitaB.org Mentorship-iOS app
//

**Additional context**
- [ ] The file contents for the above proposed headers are:
//
//  afilename.swift
//
//  Created for the AnitaB.org Mentorship-iOS app
//
- [ ] The headers must be specified in an appropriately formatted `IDETemplateMacros.plist` file in the path:
`<ProjectName>.xcodeproj/xcshareddata/` and checked into the project repo so that all contributors will be setting the same headers.
- [ ] Headers in all current swift files must be changed.

### PR DESCRIPTION
**Pull request by sunjunkie**
### Description
Set custom file headers for the project in Xcode. Edited all source files to match new headers.

Fixes #56 

### Type of Change:

- Documentation (file header comments)

### How Has This Been Tested?
Visually verified. Ran project in simulator.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the headers

### CODE REVIEW

File: mentorship ios/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -2,6 +2,6 @@
   Localizable.strings
   mentorship ios
 
-  Created by Yugantar Jain on 04/06/20.
+  Created on 04/06/20.
   Copyright © 2020 Yugantar Jain. All rights reserved.
 */
```
Comment (yugantarjain) [inline mentorship ios/ru.lproj/Localizable.strings]: Copyright line is still present.

File: mentorship ios/Extensions/UserDefaults.swift [hunk 2]
```diff
@@ -1,9 +1,7 @@
 //
-//  UserDefaults.swift
-//  mentorship ios
-//
-//  Created by Yugantar Jain on 08/06/20.
-//  Copyright © 2020 Yugantar Jain. All rights reserved.
+// UserDefaults.swift
+// Created on 08/06/20.
+// Created for AnitaB.org Mentorship-iOS
```
Comment (yugantarjain) [inline mentorship ios/Extensions/UserDefaults.swift:4]: The filename("UserDefaults.swift") is also there in the header. We have omitted that in other files. 
I think we can keep it in every file.

**Comment by vatsalkul** [comment]
Can you also edit the tests files i.e. MentorshipTests.swift, MentorshipUITests.swift?

**Review by vatsalkul — CHANGES_REQUESTED** [review]
Can you also edit the tests files i.e. MentorshipTests.swift, MentorshipUITests.swift?

File: mentorship ios/ru.lproj/Localizable.strings (hunk 1 above)
Comment (sunjunkie) [reply mentorship ios/ru.lproj/Localizable.strings]: I think ru.lproj is left over from localization testing. These files aren't actually part of the project now. I'll remove them.

**Review by vatsalkul — APPROVED** [review]

**Review by yugantarjain — APPROVED** [review]

**Pull request merged**